### PR TITLE
bescort: add ratha to fang cove mammoth connection

### PR DESCRIPTION
**Pull request by jandersson**

### CODE REVIEW

File: bescort.lic [hunk 1]
```diff
@@ -420,12 +420,18 @@ class Bescort
...
 
   def take_mammoth(mode)
-    if mode == 'fang'
-      manual_go2(2239)
-      mammoth_type = 'tall'
-    elsif mode == 'acen'
+    case mode
+    when 'fang'
+      id_to_mammoth = { 2239 => 'tall', 11130 => 'sea' }
+      closest_mammoth = sort_destinations([2239, 11130]).first
```
Comment (rpherbig) [inline bescort.lic]: What do you think about `closest_mammoth = sort_destinations(id_to_mammoth.keys).first`?

Comment (jandersson) [inline bescort.lic]: Sounds good!

**Comment by jandersson** [comment]
Note: handles #1085

File: bescort.lic [hunk 2]
```diff
@@ -418,14 +418,22 @@ class Bescort
 
     move 'climb bridge'
   end
+StringProc.new("start_script('bescort', ['mammoth', 'acen']);wait_while{running?('bescort')};")
```
Comment (rpherbig) [inline bescort.lic]: What do these two lines do?

**Pull request merged**